### PR TITLE
[feature-99] Two more files to 'pull-requests' remote repository

### PR DESCRIPTION
**Pull request by zamulaaamPaukeee**
I added two more files in folder feature-99 to 'pull-requests' remote repository:
1. new-file.txt
2. another-file.txt

Greetings from Poland! Bogdan you are the best!

### CODE REVIEW

**Review by ArniyhJS — APPROVED** [review]
great work buddy.

File: feature-99/another-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Another file to test pull request feature.
```
Comment (ArniyhJS) [inline feature-99/another-file.txt:1]: it is really good that you have decided to add this line of code

Comment (ArniyhJS) [reply feature-99/another-file.txt:1]: great job

**Review by ArniyhJS — APPROVED** [review]

**Review by mikegithubber — APPROVED** [review]
Merging. Thank you!

**Pull request merged**